client/my-sites/marketing/connections/mailchimp-settings.jsx

```diff
@@ -146,6 +169,45 @@ const MailchimpSettings = ( {
 						</option>
 					) ) }
 			</select>
+			{ !! membershipsConnectedAccountId &&
+				!! mailchimpListId &&
+				!! mailchimpLists &&
+				mailchimpLists.length > 0 && (
+					<div>
+						<p>
+							{ translate(
+								'Your site has also enabled {{a}}Recurring Payments{{/a}}.{{br/}}Do you want your Recurring Payments subscribers to be added to a Mailchimp list?',
```

Comment:
Here's how I would word this:

`Your site also has Recurring Payments enabled. Do you want your subscribers to be added to a Mailchimp list?`

Refer to the design in my prevision comment for spacing.